ui/src/message_popup/renderers/minimal_timed_view.jsx

```diff
@@ -0,0 +1,88 @@
+/* @flow weak */
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+var sceneTimer;
+
+// This response supports responding with text.
+export default React.createClass({
+  displayName: 'MinimalTimedView',
+
+  propTypes: {
+    onLogMessage: React.PropTypes.func.isRequired,
+    onResponseSubmitted: React.PropTypes.func.isRequired,
+    recordText: React.PropTypes.string,
+  },
+
+  getDefaultProps() {
+    return {
+      recordText: 'Respond',
+      autoFocus: true,
+      textHeight: 48
+    };
+  },
+
+
+  getInitialState() {
+    sceneTimer = setTimeout(this.onTimer, 25000);
+    return {
+      responseText: 'type here!'
+    };
+  },
+
+  onTextChanged(e) {
+    const responseText = e.target.value;
+    this.setState({responseText});
+  },
+
+  onTimer() {
+    const {responseText} = this.state;
+    this.props.onLogMessage('message_popup_text_response', {responseText});
+    this.props.onResponseSubmitted({ignore: true});
+  },
+
+
+  onSkipTapped() {
+    clearTimeout(sceneTimer);
+    const {responseText} = this.state;
+    this.props.onLogMessage('message_popup_text_response', {responseText});
+    this.props.onResponseSubmitted({ignore: true});
+  },
+
+
+  render() {
+    const {
+      recordText
+    } = this.props;
+   
+    return (
+      <div className="MinimalTimedView" style={styles.container}>
+        <div style={styles.buttonRow}>
+          <RaisedButton key="record" onTouchTap={this.onSkipTapped} label={recordText} secondary={true} />
+        </div>
+      </div>
+    );
+  }
+});
+
+
+const styles = {
+  container: {
+    paddingBottom: 5,
+    margin: 20
+  },
+  textField: {
```

Comment:
I think all of these styles except `container` can be deleted?  Just check if they're used in this file and if they're not used anymore delete 'em!